### PR TITLE
Add survey chains

### PR DESCRIPTION
**Pull request by hiletmis**
Related https://github.com/api3dao/tasks/issues/958

- [x] `mode`
- [ ] `mode-sepolia-testnet`
- [x] `fraxtal`
- [ ] `fraxtal-holesky-testnet`
- [ ] `linea-sepolia-testnet`

### CODE REVIEW

**Comment by bbenligiray** [comment]
See https://github.com/api3dao/tasks/issues/958#issuecomment-2051330524 specifically

**Comment by bbenligiray** [comment]
@bdrhn9 reminder to update https://github.com/api3dao/nodary-rpc-watcher after this gets released

File: .changeset/popular-walls-help.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@api3/chains": patch
+---
+
+Adds RPC providers to `mode` and `mode-sepolia-testnet` chains
```
Comment (bdrhn9) [inline .changeset/popular-walls-help.md:5]: We can remove `mode-sepolia-testnet` here.

**Review by bdrhn9 — COMMENTED** [review]
We can add `tenderly.co` as a private provider for `fraxtal`. I also couldn't find another provider for this chain. Additionally, it might be worth considering setting up a node for this chain in the future.

**Review by bdrhn9 — APPROVED** [review]

**Pull request merged**